doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,145 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Using design parameters
+
+You can read and update parameters that are part of the design.
+In this example, we have a simple design with two associated parameters.
```

Comment:
```suggestion
The simple design in this example has two associated parameters.
```
According to our style guide (Google developer documentation style guide), avoid "we" and write in active voice. Is the word "associated" needed?